src/util/invariant.h

```diff
@@ -132,6 +133,15 @@ class invariant_with_diagnostics_failedt : public invariant_failedt
   }
 };
 
+#ifdef __GNUC__
+#define CBMC_NORETURN __attribute((noreturn))
+#elif defined(_MSV_VER)
+#define CBMC_NORETURN __declspec(noreturn)
+#elif __cplusplus >= 201703L
+#define CBMC_NORETURN [[noreturn]]
+#else
+#define CBMC_NORETURN
+#endif
 // The macros MACRO<n> (e.g., INVARIANT2) are for internal use in this file
 // only. The corresponding macros that take a variable number of arguments (see
 // further below) should be used instead, which in turn call those with a fixed
```

Comment:
These comments can be removed, and the lines following right after that still contain the macros `INVARIANT3`, etc. should be adapted to the new macros.